winappdbg/win32/user32.py

```diff
@@ -71,7 +71,7 @@ class __WindowEnumerator (object):
     def __init__(self):
         self.hwnd = list()
     def __call__(self, hwnd, lParam):
-##        print hwnd  # XXX DEBUG
+##        print(hwnd  # XXX DEBUG)

```

Comment:
OMG so broken hahaha

I get it, automated porting tools are hard, but why try to parse what's inside comments anyway?